### DISCUSSION TITLE
add second display to VM, only blank display

### DISCUSSION THREAD

**Discussion by PetaHub** [utmapp/UTM#6136] [UTM Help (macOS)]
I need some help to configure a second display. The VM is a Debian 12 VM (ARM).

The primary (single) display is => Emulated Display = Card virtio-ramb-gl (GPU Supported).

I have added a second display => Emulated Display Card = secondary-vga

The VM starts without an error message, the primary display works without problems, but the secondary display remains dark and is not recognized in the system settings under the Debian VM.

I have tried all possible display settings under the VM settings, unfortunately no combination has produced a positive result.   

I am happy for any help or suggestions. 🙂

Configuration
UTM Version:  Version 4.4.5 (94)
macOS Version: macOS 14.2.1 (23C71)
Mac Chip: M1, MacBook Air M1

**Comment by osy** [comment]
Okay I know this is stupid but "secondary-vga" doesn't work for ARM. It only works on x86 VMs. You need to set the second display to "ramfb" or "virtio-gpu-pci" or something.
